Packs/Azure-Enrichment-Remediation/Playbooks/Azure_-_Network_Security_Group_Remediation.yml

```diff
@@ -672,7 +672,7 @@ tasks:
                       iscontext: true
                     right:
                       value:
-                        simple: ${inputs.AzureVMPrivateIP}
+                        simple: inputs.AzureVMPrivateIP
```

Comment:
I think here it missing the `iscontext` key. Can you please check within the UI if that is the case?

Reply:
[4e8df20](https://github.com/demisto/content/pull/35650/commits/4e8df20444cb58214f1f33548297dd6ca739fa10)